app/script/notification/NotificationRepository.js

```diff
@@ -209,7 +212,7 @@ z.notification.NotificationRepository = class NotificationRepository {
     if (messageEntity.has_asset_text()) {
       for (const assetEntity of messageEntity.assets()) {
         if (assetEntity.is_text()) {
-          const notificationText = assetEntity.isSelfMentioned()
+          const notificationText = assetEntity.isUserMentioned(this.userRepository.self().id)
```

Comment:
We use self three times already in this file and it causes a line to break. Fine to keep a reference by adding `this.selfUser = this.userRepository.self`